acquia/acquia_cms
# ACMS-1030: Update search api and search api solr module to latest

**Motivation**

Fixes #[ACMS-1030](https://backlog.acquia.com/browse/ACMS-1030)

**Proposed changes**

Update search api and search api solr module to latest

**Alternatives considered**

NA

**Testing steps**


- Install site with this branch
- Visit module page and verify search api and search api solr module are update.
- Enable starte module all page.

**Merge requirements**
- [_Enhancement_] _Major change_, _Minor change_, _Bug_, _Enhancement_, and/or _Chore_ label applied
